pyproject.toml

```diff
@@ -56,6 +56,7 @@ docs = [
 
 [tool.hatch]
```

Comment:
This it hatch setup stuff but hatchling (the backend) handles the vcs part. So how does sphinx builder work? in this case the version is not being recognized via the new tags which is why the release doesn't work.